apps/i3wm/i3wm.talon

```diff
@@ -2,93 +2,83 @@
+resize mode: user.system_command("i3-msg mode \"resize\"")
 focus parent: user.system_command("i3-msg focus parent")
 focus child: user.system_command("i3-msg focus child")
-
 # resize helpers
 grow window:
-    user.system_command('i3-msg mode "resize"')
+    user.system_command("i3-msg mode \"resize\"")
     key(right:10)
     key(down:10)
     # escape resize mode
     key(escape)
     # center window
     sleep(200ms)
     user.system_command("i3-msg move position center")
-
-
 # resize helpers
 shrink window:
-    user.system_command('i3-msg mode "resize"')
+    user.system_command("i3-msg mode \"resize\"")
     key(left:10)
     key(up:10)
     # escape resize mode
     key(escape)
     # center window
     sleep(200ms)
     user.system_command("i3-msg move position center")
-
-horizontal (shell|terminal):
+horizontal (shell | terminal):
     user.system_command("i3-msg split h")
     user.i3wm_shell()
-
-vertical (shell|terminal):
+vertical (shell | terminal):
     user.system_command("i3-msg split v")
     user.i3wm_shell()
-
 # XXX - just replace with shuffle eventually?
 # XXX - like also need to match the generic talon commands
-(shuffle|move (win|window) [to] port) <number_small>:  user.system_command("i3-msg move container to workspace number {number_small}")
-(shuffle|move (win|window) [to] port ten): user.system_command("i3-msg move container to workspace number 10")
-(shuffle|move (win|window) [to] last port): user.system_command("i3-msg move container to workspace back_and_forth")
-(shuffle|move (win|window) left): user.system_command("i3-msg move left")
-(shuffle|move (win|window) right): user.system_command("i3-msg move right")
-(shuffle|move (win|window) up): user.system_command("i3-msg move up")
-(shuffle|move (win|window) down): user.system_command("i3-msg move down")
-
-(win|window) horizontal: user.system_command("i3-msg split h")
-(win|window) vertical: user.system_command("i3-msg split v")
-
+(shuffle | move (win | window) [to] port) <number_small>:
+    user.system_command("i3-msg move container to workspace number {number_small}")
+(shuffle | move (win | window) [to] port ten):
+    user.system_command("i3-msg move container to workspace number 10")
+(shuffle | move (win | window) [to] last port):
+    user.system_command("i3-msg move container to workspace back_and_forth")
+(shuffle | move (win | window) left): user.system_command("i3-msg move left")
+(shuffle | move (win | window) right): user.system_command("i3-msg move right")
+(shuffle | move (win | window) up): user.system_command("i3-msg move up")
+(shuffle | move (win | window) down): user.system_command("i3-msg move down")
+(win | window) horizontal: user.system_command("i3-msg split h")
+(win | window) vertical: user.system_command("i3-msg split v")
 make scratch: user.system_command("i3-msg move scratchpad")
-[(show|hide)] scratch: user.system_command("i3-msg scratchpad show")
+[(show | hide)] scratch: user.system_command("i3-msg scratchpad show")
```

Comment:
Would be good if talonfmt could get rid of the unnecessary parentheses here, `[show | hide] scratch: ...`

Reply:
Should be possible, but I'll put that on the future work backburner, currently it preserves parentheses exactly.

Reply:
Filed this as an issue for future work.